main.py

```diff
@@ -372,35 +373,47 @@ def user_token(self, userid='core'):
     
     def check_docker(self):
         '''Check docker services status'''
-        dockercloud.user = self.config['DOCKERCLOUD_USER']
-        dockercloud.apikey = self.config['DOCKERCLOUD_APIKEY']
-        dockercloud.namespace = self.config['DOCKERCLOUD_NAMESPACE']
-        d = dockercloud.Service.list()
-        for service in d:
-            print(service.name + " : " + service.state)
-            
+        stack = "docker-cloud stack inspect datahub-production"
+        try:
+            process= subprocess.Popen(stack.split(), stdout=subprocess.PIPE)
+            output, error = process.communicate()
+            output = output.split(',')
+            print(output[5])
+        except:
+            print("Command not found")
+        
     def check_apis(self):
-        ''' Check API call for all services'''
+        '''Check API call for all services'''
         valid_token = self.config['JWT_TOKEN']
         invalid_token = 'ivalid_token'
-        api_base_url = 'http://' + self.config['DOMAIN_API'] 
+        api_base_url = 'http://api.' + self.config['DOMAIN_BASE'] 
         # Auth service
         url_auth_authorize = urljoin(api_base_url,'auth/authorize')
         response = requests.get(url_auth_authorize)
-        assert (response.status_code == 200)
-        print("Get permission for a service - successfull connected")
-        url_auth_check = urljoin(api_base_url, 'auth/check')
-        response = requests.get(url_auth_check, headers={'Auth-Token': valid_token})
-        assert (response.status_code == 200) 
-        if "true" in response.text:
-            print("Check an authentication token's validity for VALID token - successfull connected")
-        
+        try:
+            assert (response.status_code == 200)
+            print("Get permission for a service - successfull connected")
+        except:
+            print("Get permission for a service is unavailable")
+        if valid_token:
+            url_auth_check = urljoin(api_base_url, 'auth/check')
+            response = requests.get(url_auth_check, headers={'Auth-Token': valid_token})
+            try:
+                assert (response.status_code == 200) 
```

Comment:
I would get rid of this `Try/assert` thing. It should fail if can not pass assertion
You can use `assert(response.status_code == 200, message_goes_here)` if you want to say something when failed